pipeline_dp/private_spark.py

```diff
@@ -12,10 +13,32 @@ class PrivateRDD:
     within the specified privacy budget can be extracted from it through its API.
```

Comment:
Please add the following to class comment:

`PrivateRDD` keeps `privacy_id` for each element in order to guarantees correct DP computations.

Reply:
added